app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
+end
```

Comment:
- [ ] In this file, it is expected that the Post model has a method that updates the posts counter for a user. Kindly add that 👍🏽
- [ ] I will also encourage you to have  the `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private
- [ ] It is required that the Post model has a method that returns the 5 most recent comments for a given post. Kindly work on these 👍🏽

```ruby
after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

**It is nice if you could add the after_save to the other files as well for any update made**